### DISCUSSION TITLE
我的clash meta for android订阅错误

### DISCUSSION THREAD

**Discussion by qazxcgy** [MetaCubeX/ClashMetaForAndroid#206] [General]
日志显示是
[TCP] mihomo --> xn--cp3a08l.com:443 doesn't match any rule using DIRECT
哪位大佬知道什么意思嘛
![S40410-184104(1)](https://github.com/MetaCubeX/ClashMetaForAndroid/assets/139515302/b6f11c15-ceed-4bb8-9724-e449ee91c980)

**Comment by lim-kim930** [comment]
这并不是保错，日志内容就是规则没匹配上xn--cp3a08l.com，所以走了直连而已，你说的保错是指？

**Reply by qazxcgy** [reply]
问一下，这个问题怎么办，大佬
![S40408-222552(1)](https://github.com/MetaCubeX/ClashMetaForAndroid/assets/139515302/81a72b24-25ff-431b-8851-fea264488438)

**Reply by lim-kim930** [reply]
貌似这网站的证书过期了，不过我刚刚试了下这个域名正常可以访问了
不过你这是刚截的图吗？要是的话你这个本地时间不太对啊，你要不再导入一次试试？
